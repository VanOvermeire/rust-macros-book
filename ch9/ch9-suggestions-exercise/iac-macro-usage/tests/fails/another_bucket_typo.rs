use iac_macro::iac;

fn main() {
    iac! {
        buck uniquename
    }
}
