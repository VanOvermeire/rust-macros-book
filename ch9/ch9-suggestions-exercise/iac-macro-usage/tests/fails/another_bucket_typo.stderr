error: only 'bucket' and 'lambda' resources are supported. Is this a typo for bucket?
 --> tests/fails/another_bucket_typo.rs:5:9
  |
5 |         buck uniquename
  |         ^^^^
